examples/entity-viewer-v2/entity-viewer-client/src/main/ml-modules/root/explore-data/ui-config/config.json

```diff
@@ -315,6 +315,11 @@
       "component": "ResultsList",
       "config": {
         "pageLengths": [3, 5, 10, 20],
+        "sort":{
```

Comment:
When I remove the sorting config object, I still see the sorting arrows:
<img width="724" alt="Screen Shot 2022-04-25 at 11 16 41 AM" src="https://user-images.githubusercontent.com/477757/165149709-6e3f3a3b-8922-49ee-ab70-407fa0d3a6b1.png">
Then when I click the arrows, I get an error:
<img width="705" alt="Screen Shot 2022-04-25 at 11 17 17 AM" src="https://user-images.githubusercontent.com/477757/165149809-5f41b47f-dd22-49ee-993e-0bd067f30c70.png">
Expected behavior, if the config.json results object is missing a sort configuration object, then the user should see NO label and NO arrows and there should be no clickable sort in the UI.